qpmodel/optimizer.cs

```diff
@@ -50,8 +51,67 @@ public class Property { }
     public class PhysicProperty : Property
     {
         // ordering: the ordered expression and whether is descending
-        KeyValuePair<Expr, bool> ordering_;
-        KeyValuePair<Expr, DistributionType> distribution_;
+        List<KeyValuePair<Expr, bool>> ordering_;
+        List<KeyValuePair<Expr, DistributionType>> distribution_;
+        public PhysicProperty(List<Expr> order, List<bool> desc = null)
```

Comment:
https://coveralls.io/builds/31922927/source?filename=qpmodel/optimizer.cs

Seems these code are not exercised in unittest, can you add them?

Reply:
Sure

Reply:
also, consider removing distribution_ and introducing it in your next PR with unittest.